src/SvelteTable.svelte

```diff
@@ -174,5 +221,40 @@
     {:else}
       <slot name="empty" />
     {/if}
+    {#if hasPagination}
+      <tr>
```

Comment:
I don't particularly appreciate that the pagination is the part of the table. This is not the content. I suggest moving it outside of the table

Reply:
What if the rows on the page are less than 0 or rowsPerPage? Should we don't display it/disable pagination

Reply:
Also, as pagination is not a real part of the table, I suggest moving it in the separate component and add unit tests for it